funmi5/PropertyPro-Lite
# #166839971 chore: Add and edit some UI files

**What does this PR do?**
Designing the specific page and editing some pages for PropertyPro-Lite

**Description of task**
- add pictures for the view properties and specific property pages
- add js codes for the navigation sidebars
- add icons for the user dashboard
- design specific property page(prop1) with html and css

**How should this be manually tested/checked?**
<a href= "https://funmi5.github.io/PropertyPro-Lite/UI">Click on this link to check the web page</a>

**What are the relevant pivotal tracker stories?**
<a href= "https://www.pivotaltracker.com/n/projects/2355287">#166839971</a>
